roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -48,3 +48,6 @@
     - name: Ceph cluster status
       ansible.builtin.debug:
         msg: "{{ ceph_health.stdout_lines }}"
+
+- name: Dashabord service validation
+  ansible.builtin.include_tasks: dashboard/validation.yml
```

Comment:
@katarimanojk  would you please add a `when` on line 54 so that this validation only runs if the user has set this?
```
cifmw_ceph_daemons_layout:
  dashboard_enabled: true
```
Do we have ci logs of the output of this job? I'd like to examine `podified-multinode-hci-deployment-crc` to see the changed code run.  I think we're close to merging we just need those things IMO.

Reply:
thanks.